src/platform_impl/macos/app_state.rs

```diff
@@ -79,36 +80,38 @@ where
 }
 
 #[derive(Default)]
-struct Handler {
+struct Handler<'a> {
     ready: AtomicBool,
     in_callback: AtomicBool,
     control_flow: Mutex<ControlFlow>,
     control_flow_prev: Mutex<ControlFlow>,
     start_time: Mutex<Option<Instant>>,
     callback: Mutex<Option<Box<dyn EventHandler>>>,
-    pending_events: Mutex<VecDeque<Event<Never>>>,
-    deferred_events: Mutex<VecDeque<Event<Never>>>,
+    pending_events: Mutex<VecDeque<Event<'a, Never>>>,
+    // We don't need it any more. See line 270
```

Comment:
Please don't refer to things by line number. Those are constantly changing!

Reply:
@ZeGentzy These are PR only, I'll remove them before it's merged.